src/Pages/Form/Components/PatientInformationSection.tsx

```diff
@@ -0,0 +1,265 @@
+import "date-fns";
+import React, { useState } from "react";
+import {
+  createStyles,
+  makeStyles,
+  withStyles,
+  Theme,
+  ThemeProvider,
+  createTheme,
+} from "@material-ui/core/styles";
+import Box from "@material-ui/core/Box";
+import Grid from "@material-ui/core/Grid";
+import TextField from "@material-ui/core/TextField";
+import Typography from "@material-ui/core/Typography";
+import Button from "@material-ui/core/Button";
+import MaleIcon from "../../../Assets/male.png";
+import FemaleIcon from "../../../Assets/female.png";
+import DateFnsUtils from "@date-io/date-fns";
+import {
+  MuiPickersUtilsProvider,
+  KeyboardTimePicker,
+  KeyboardDatePicker,
+} from "@material-ui/pickers";
+import ToggleButton from "@material-ui/lab/ToggleButton";
+import ToggleButtonGroup from "@material-ui/lab/ToggleButtonGroup";
+
+const theme = createTheme({
+  palette: {
+    primary: {
+      main: "#FFF",
+      contrastText: "#3A3A3D",
+    },
+    secondary: {
+      main: "#EF5DA8",
+      contrastText: "#FFF",
+    },
+  },
+});
+
+const useStyle = makeStyles((theme) =>
+  createStyles({
+    root: {
+      width: "100%",
+      marginLeft: "32px",
+    },
+    picker: {
+      marginTop: "34px",
+    },
+    textField: {
+      width: "80%",
+    },
+    textTitle: {
+      marginTop: "40px",
+    },
+    boxMale: {
+      width: "165px",
+      textAlign: "center",
+    },
+    boxFemale: {
+      width: "190px",
+      textAlign: "center",
+    },
+    button: {
+      "&.Mui-selected": {
+        backgroundColor: "#EF5DA8",
+        color: "#FFF",
+        pointerEvents: "none",
+      },
+    },
+    buttonUpload: {
+      textTransform: "none",
+      marginTop: "16px",
+      color: "#FFF",
+      backgroundColor: "#3A3A3D",
+      boxShadow: "0px 2px 5px rgba(0, 0, 0, 0.1)",
+      borderRadius: "20px",
+      width: "170px",
+      height: "40px",
+    },
+  })
+);
+
+const GenderStyledToggleButtonGroup = withStyles((theme) => ({
+  grouped: {
+    margin: theme.spacing(3),
+    boxShadow: "0px 2px 4px rgba(0, 0, 0, 0.1)",
+    width: "150px",
+    height: "95px",
+    backgroundColor: "#FFF",
+    border: "none",
+    "&:not(:first-child)": {
```

Comment:
Why did you set both not first child and first child ?

Reply:
If you want to set borderRadius for all children, you can use child selector or create new style and set it to child directly.